lib/BackgroundJobs/AdminNotification.php

```diff
@@ -33,7 +33,7 @@ protected function run($argument) {
 			->setDateTime(new \DateTime())
 			->setSubject('updated')
 			->setObject('dummy', 23)
-			->setLink($urlGenerator->getAbsoluteURL('index.php/settings/admin#usage-report'));
+			->setLink($urlGenerator->getAbsoluteURL('index.php/settings/admin/survey_client'));
```

Comment:
you can also use linkToRoute() instead, makes you robust against possible future url changes.